quicktile/commands.py

```diff
@@ -167,9 +167,9 @@ def decorate(func):
             return func
         return decorate
 
-    def call(self, command, winman, *args, **kwargs):
+    def check_command(self, command, winman, *args, **kwargs):
+        """ check if the command is valid and execute it"""
```

Comment:
The name `check_command` implies that it won't execute the command if it does exist. (A view supported by the docstring saying "check ... and execute it".)

Please either leave this as `call` or rename it to `try_call`.